controller_manager/scripts/spawner

```diff
@@ -191,9 +200,6 @@ def main():
 
     rospy.loginfo("Controller Spawner: Loaded controllers: %s" % ', '.join(loaded))
 
-    if rospy.is_shutdown():
```

Comment:
whiy did you remove this block?

Reply:
It seemed redundant since a shutdown hook is defined, and `rospy.spin()` simply keeps python from exiting until `rospy.core.is_shutdown()` is set (see [rospy/client.py](https://github.com/ros/ros_comm/blob/kinetic-devel/clients/rospy/src/rospy/client.py#L118)).